<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>manageWindow</class>
 <widget class="QWidget" name="manageWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>管理界面</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item alignment="Qt::AlignLeft">
    <widget class="QPushButton" name="changeUserButton">
     <property name="text">
      <string>切换用户</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="3,1,2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>添加用户</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignLeft">
        <widget class="QPushButton" name="addUserButton">
         <property name="text">
          <string>添加用户</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3,4">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>用户名</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="addUserNameEdit">
           <property name="accessibleDescription">
            <string/>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="inputMask">
            <string/>
           </property>
           <property name="placeholderText">
            <string>用户名</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,3,1,3">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>密码</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="addUserPasswordEdit">
           <property name="placeholderText">
            <string>密码</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>重复密码</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="addUserPasswordEdit_2">
           <property name="placeholderText">
            <string>重复密码</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>修改用户密码</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignLeft">
        <widget class="QPushButton" name="changePasswordButton">
         <property name="text">
          <string>更改密码</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,3,4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>用户名</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="changeUserNameEdit">
           <property name="placeholderText">
            <string>用户名</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,3,1,3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>新密码</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="changeUserPasswordEdit">
           <property name="placeholderText">
            <string>新密码</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>重复密码</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="changeUserPasswordEdit_2">
           <property name="placeholderText">
            <string>重复新密码</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>删除用户</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="queryUserButton">
           <property name="text">
            <string>查询用户</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteUserButton">
           <property name="text">
            <string>删除用户</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,3,4">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>待删除用户名</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="deleteUserEdit">
           <property name="placeholderText">
            <string>用户名</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="Line" name="line_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item alignment="Qt::AlignRight">
        <widget class="QPushButton" name="clearManageInfoShowButton">
         <property name="text">
          <string>清除信息</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="manageInfoShow"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
